vulture_os/toolkit/api_parser/netskope/netskope.py

```diff
@@ -129,6 +129,9 @@ def execute(self):
             self.update_lock()
 
             logs = response['result']
+            if len(logs) == 0:
+                self.frontend.last_api_call = to
```

Comment:
Please remove this line, last_api_call is handled just below.
We don't want to update last_api_call is there is no logs.